src/components/general/Button/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { type FC } from 'react'
+import { InvertColourButton } from './styles'
+
+export const Button: FC = () => {
+  return (
+    <div>
```

Comment:
We dont need to have this random div here if it does nothing, just return the invertcolourbutton